Multiworld: 2 Nintendont connections for 1 player not working

Tested on Randovania 6.1.0.dev160

Solo multiworld:
 -Row 1: Prime 1 on Wii U vWii Nintendont
 -Row 2: Echoes on Wii Nintendont

Both show as connected and in-game/title screen. The tracker for Prime 1 shows current inventory and shows the correct number of collected locations in the session window. The tracker for Echoes shows no items or collected locations and doesn't send/receive any items.

This is spammed in the log every 3 seconds
```
[2023-07-09 01:45:58,147] [ERROR] [randovania.gui.qt] catch_exceptions_async: Exception in callback InfiniteTimer._on_timeout.<locals>._error_handler(<Task finishe...ent server.')>) at /home/ethan/randovania/randovania/lib/infinite_timer.py:40
Traceback (most recent call last):
  File "/usr/lib/python3.11/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/home/ethan/randovania/randovania/lib/infinite_timer.py", line 43, in _error_handler
    return t.result()
           ^^^^^^^^^^
  File "/home/ethan/randovania/randovania/lib/infinite_timer.py", line 34, in _target_wrap
    return await self.target()
           ^^^^^^^^^^^^^^^^^^^
  File "/home/ethan/randovania/randovania/game_connection/connector/prime_remote_connector.py", line 303, in update
    await self.update_current_inventory()
  File "/home/ethan/randovania/randovania/game_connection/connector/prime_remote_connector.py", line 322, in update_current_inventory
    new_inventory = await self.get_inventory()
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ethan/randovania/randovania/game_connection/connector/echoes_remote_connector.py", line 113, in get_inventory
    arr_raw = await self.executor.perform_single_memory_operation(MemoryOperation(
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ethan/randovania/randovania/game_connection/executor/memory_operation.py", line 61, in perform_single_memory_operation
    result = await self.perform_memory_operations([op])
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ethan/randovania/randovania/game_connection/executor/nintendont_executor.py", line 228, in perform_memory_operations
    requests = self._prepare_requests_for(ops)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ethan/randovania/randovania/game_connection/executor/nintendont_executor.py", line 191, in _prepare_requests_for
    raise ValueError(f"Request {op} is not compatible with current server.")
ValueError: Request At *0x803dbfb0 +0000, read 128 bytes is not compatible with current server.
```

--- a/randovania/game_connection/connector/echoes_remote_connector.py
+++ b/randovania/game_connection/connector/echoes_remote_connector.py
@@ -110,12 +110,17 @@ class EchoesRemoteConnector(PrimeRemoteConnector):
         # mapWorldInfoAreas.areas: + 0x4
         # mapWorldInfoAreas.areas.data: +0xc
 
-        arr_raw = await self.executor.perform_single_memory_operation(MemoryOperation(
-            address=self.version.cstate_manager_global + 0x8c0 + 0x4 + 0xc,
-            read_byte_count=4 * 32,
-            offset=0,
-        ))
-        arr = struct.unpack(">32L", arr_raw)
+        # multiple passes are required, as 128 bytes is too much at once for Nintendont
+        PASSES = 2
+        arr_raws = [
+            await self.executor.perform_single_memory_operation(MemoryOperation(
+                address=self.version.cstate_manager_global + 0x8c0 + 0x4 + 0xc,
+                read_byte_count=4 * (32 // PASSES),
+                offset=i * 4 * (32 // PASSES),
+            ))
+            for i in range(PASSES)
+        ]
+        arr = struct.unpack(">32L", b''.join(arr_raws))
 
         count = 0
 

--- a/test/game_connection/connector/test_echoes_remote_connector.py
+++ b/test/game_connection/connector/test_echoes_remote_connector.py
@@ -75,7 +75,7 @@ async def test_get_inventory_valid(connector: EchoesRemoteConnector):
         op: struct.pack(">II", item.max_capacity, item.max_capacity)
         for op, item in zip(ops, connector.game.resource_database.item)
     }
-    connector.executor.perform_single_memory_operation.return_value = b"\x00\x00\x00\x00" * 32
+    connector.executor.perform_single_memory_operation.return_value = b"\x00\x00\x00\x00" * 16
     _override = {
         "ObjectCount": 0,
     }

